cpp/thirdparty/versions.txt

```diff
@@ -24,7 +24,7 @@
 # format).
 
 ARROW_ABSL_BUILD_VERSION=2eba343b51e0923cd3fb919a6abd6120590fc059
-ARROW_AWSSDK_BUILD_VERSION=1.7.160
+ARROW_AWSSDK_BUILD_VERSION=1.8.57
```

Comment:
Works with the previous version as well.

Reply:
We should bump to 1.8 though because of improvements to AWS region detection